spec.html

```diff
@@ -13480,13 +13480,15 @@ <h1>
 
       <emu-clause id="sec-%throwtypeerror%">
         <h1>%ThrowTypeError% ( )</h1>
-        <p>The <dfn>%ThrowTypeError%</dfn> intrinsic is an anonymous built-in function object that is defined once for each realm. When %ThrowTypeError% is called it performs the following steps:</p>
+        <p>This function is the <dfn>%ThrowTypeError%</dfn> intrinsic object.</p>
```

Comment:
“intrinsic object” seems redundant here; the % syntax already conveys that.

Reply:
So would you prefer
```This function is <dfn>%ThrowTypeError%</dfn>.```
? And likewise for the other function properties of the global object?

The words "intrinsic object" are certainly redundant, but they might be useful. Imagine someone is browsing the spec, and they don't know about intrinsic objects, but they come across one of these percent-thingies, so they click on it and land at a clause that says "This function is %foo%." Well, that's something, but it doesn't explain what a percent-thingy is. However, saying "This function is the %foo% intrinsic object" would give them a phrase that they could then look up. (It would be better if "intrinsic object" were an auto-linked phrase.)

Reply:
Yes, that’d be what i expect.

% syntax should ideally link to the section describing it, yes.

Reply:
> % syntax should ideally link to the section describing it, yes.

Are you saying that uses of the % syntax (e.g. %Foo%) should link to [6.1.7.4 Well-Known Intrinsic Objects](https://tc39.es/ecma262/#sec-well-known-intrinsic-objects) rather than the clause where the particular %Foo% is defined?

Reply:
Hmm - I’m just saying that the syntax should link somewhere; i don’t have a strong opinion as to where.

Reply:
Okay, well:
- Every use of a well-known intrinsic already auto-links to the clause that defines it.
- Ditto for the bulk of the not-well-known (NWK) intrinsics that the spec actually references (e.g. `%Object.prototype%` and `%Function.prototype%`).
- I count 15 in-spec references to 11 NWK intrinsics that don't get auto-linked (e.g., 3 uses of `%Array.prototype.values%`). So there's room for improvement.
- And then there are scads of NWK intrinsics that wouldn't get auto-linked if they were referenced, but the spec never does (e.g. `%String.prototype.at%`).

Reply:
ok, so maybe usage should link to definition, but the definition site should link to the section explaining the syntax?

Reply:
So, e.g.,
`This function is the <dfn>%ThrowTypeError%</dfn> intrinsic object.`
could retain the redundant "intrinsic object", but that phrase could auto-link to 6.1.7.4.